syntax = "proto3";

package pyServer;

service MyService {

  rpc Add(AddRequest) returns (AddResponse) {}
  rpc PrintHelloWorld(Empty) returns (HelloWorldResponse) {}
}

message AddRequest {
  int32 a = 1;
  int32 b = 2;
}

message AddResponse {
  int32 result = 1;
}

message Empty {
  
}

message HelloWorldResponse {
  string message = 1;
}
